.github/nightly_matrix.php

```diff
@@ -91,6 +91,33 @@ function get_windows_matrix_include(array $branches) {
     return $jobs;
 }
 
+function get_macos_matrix_include(array $branches) {
+    $jobs = [];
+    foreach ($branches as $branch) {
+        foreach([true, false] as $debug) {
+            foreach([true, false] as $zts) {
+                $jobs[] = [
+                    'branch' => $branch,
+                    'debug' => $debug,
+                    'zts' => $zts,
+                    'os' => $branch === 'master' ? '13' : '12',
+                    'arch' => 'X32',
+                ];
+                if($branch['name'] === 'master') {
```

Comment:
It might make sense to compare the version instead, so that this doesn't need to be updated continuously.

Reply:
I have replaced it with a check against `$branch['version']['minor']` to run for 8.4